certification/internal/shell/has_license_test.go

```diff
@@ -0,0 +1,58 @@
+package shell
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+)
+
+var _ = Describe("HasLicense", func() {
+	var (
+		HasLicense HasLicenseCheck
+		fakeEngine cli.PodmanEngine
+	)
+
+	BeforeEach(func() {
+		fakeEngine = FakePodmanEngine{
+			RunReportStdout: `/licenses`,
+			RunReportStderr: "",
+		}
+	})
+	Describe("Checking if license can be found", func() {
+		Context("When license is found", func() {
+			BeforeEach(func() {
+				podmanEngine = fakeEngine
+			})
+			It("should pass Validate", func() {
+				ok, err := HasLicense.Validate("dummy/image")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ok).To(BeTrue())
+			})
+		})
+		Context("When license is not found", func() {
+			BeforeEach(func() {
```

Comment:
Please add a test when the directory exists, but it empty (basically, stdout would be "")